### PR TITLE
py-multidict: new port, version 4.3.1

### PR DESCRIPTION
**Pull request by IPGlider**
###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.13.5 17F77
Xcode 9.4.1 9F2000

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

File: python/py-multidict/Portfile [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+platforms           darwin
+license             Apache-2
+
+python.versions     34 35 36
+
+maintainers         {ipglider.org:miguel @ipglider} openmaintainer
+
+description         Multidict implementation.
+long_description    Multidict is dict-like collection of key-value pairs where \
+                    key might be occurred more than once in the container.
```
Comment (pmetzger) [inline python/py-multidict/Portfile]: Grammar isn't right. Perhaps "where a key might occur more than once in the container"?

Comment (IPGlider) [reply python/py-multidict/Portfile]: I just copied the description from the original documentation. I will change it.

Comment (pmetzger) [reply python/py-multidict/Portfile]: Do a pull request upstream, too. :)

**Comment by pmetzger** [comment]
Merged! Thank you, @IPGlider!

**Pull request merged**